manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
+    "angular": {
+      "version": "1.4.7",
```

Comment:
There probably ought to be some test somewhere that checks that the version is a substring of the path. Thoughts?